src/common/options/rgw.yaml.in

```diff
@@ -954,6 +954,35 @@ options:
   - '1'
   - none
   with_legacy: true
+- name: rgw_keystone_scope_enabled
+  type: bool
+  level: advanced
+  desc: Enable logging of Keystone scope information in ops log
+  long_desc: When enabled, operations authenticated via Keystone will include project,
+    user, domain, and role information in the ops log. Requires rgw_enable_ops_log
+    to be enabled.
+  default: false
```

Comment:
is this off by default for the same privacy reasons as rgw_keystone_scope_include_user?